pkg/config/validation/validation.go

```diff
@@ -1296,6 +1304,18 @@ func validateTLS(settings *networking.ClientTLSSettings) (errs error) {
 		}
 	}
 
+	if settings.Mode == networking.ClientTLSSettings_ISTIO_MUTUAL {
+		if settings.ClientCertificate != "" {
```

Comment:
This violates API backwards compatibility. A previously valid Destinationrule will now be rejected. This causes issues on upgrade, as a configuration will suddenly start being rejected.

In general, we cannot make validation stricter after an API has been introduced.

Reply:
The added part of the check for tls settings.Mode has been deleted. It is difficult to balance , in order to be compatible with the old version, users are allowed to use non-standard configurations in the new version.